spec/lib/tasks/adjust_fix_expired_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "adjust:fix_expired" do
+  before do
+    load "spec/lib/tasks/adjust_fix_expired_data.rb"
+  end
+
+  let(:klass) do
+    Class.new(ApplicationRecord) do
+      self.table_name = :processing_times
```

Comment:
why are we using this view for the test?

Reply:
Because it is the reason I was able to find this bug (The processing time was incorrectly coming back as 0 so it felt right to use in the the check as it proves the issue is fixed.